Dockerfile

```diff
@@ -16,4 +16,6 @@ RUN python3 -m venv ${POETRY_HOME} &&\
     ${POETRY_HOME}/bin/pip install poetry==${POETRY_VERSION} &&\
     poetry config virtualenvs.create false
 
+COPY ./tests/tests.py /usr/src/app/tests.py
```

Comment:
We do not want to make this part of our Dockerfile. We should either create a separate build target, or bind mount it, when we `docker run`. I suggest the former.